pkgs/development/compilers/passerine/default.nix

```
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "passerine";
-  version = "0.9.2";
+  version = "0.9.3";
 
   src = fetchFromGitHub {
     owner = "vrtbl";
     repo = "passerine";
-    rev = "dd8a6f5efc5dcb03d45b102f61cc8a50d46e8e98";
-    sha256 = "sha256-/QzqKLkxAVqvTY4Uft1qk7nJat6nozykB/4X1YGqu/I=";
+    rev = "30757a528c1656c15ed3403d355d2b9dee028991";
```

Maybe we can reuse version again?

```suggestion
    rev = "v${version}";
```

---

Hi, I don't really have the time to check whether this breaks something - have you tested this?

---

sha256 should remain the same so I don't think this will break something...